tests/manage/mcg/test_multi_region.py

```diff
@@ -15,6 +16,7 @@
 
 
 @aws_platform_required
+@skipif_ocs_version("<4.5")
```

Comment:
isn't it enough to skip with the BZ decorator added below?

Reply:
Since the bug is already ON_QA, the BZ decorator will continue to run it on all OCS versions, although it's only fixed in 4.5 and above.
I moved the OCS version decorator to the second test since the first one isn't bugged.